features/docs/cli/dry_run.feature

```diff
@@ -84,3 +84,34 @@ Feature: Dry Run
       1 step (1 undefined)
 
       """
+
+  Scenario: With BeforeAll and AfterAll hooks
+    Given a file named "features/test.feature" with:
+      """
+      Feature:
+        Scenario:
+          Given this step fails
+      """
+    And the standard step definitions
+    And a file named "features/step_definitions/support.rb" with:
```

Comment:
Can we also add (For my benefit), `Given "this step fails" { raise "Step fails" }` just to cover my OCD of having 2 diff issues

Reply:
This is actually part of "the standard step definitions"
I've copy/paste that from a scenario before that one actually

As this is not the purpose of this scenario, we could replace that with "this step pass" if you want ?

Reply:
Yes please do. That way we're not testing two things but one